src/frontend/components/injector-tree/injector-tree.html

```diff
@@ -4,34 +4,41 @@ <h4 class="primary-color pt1 pb2 m0" [hidden]="selectedNode">
       No component selected
     </h4>
   </div>
-
-  <div class="flex-column monospace pl4 pt2"
+  <div class="flex-column monospace pt2"
     [hidden]="!selectedNode"
     [ngClass]="{flex: selectedNode}">
-    <div *ngIf="parentHierarchyDisplay && parentHierarchyDisplay.length > 0">
```

Comment:
You can use the safe navigation operator `?.` to replace the `&&` shortcut:

This will suffice:
```js
parentHierarchyDisplay?.length > 0
```